salt/modules/cmdmod.py

```diff
@@ -354,15 +358,11 @@ def _run(
         # The last item in the list [-1] is the current method.
         # The third item[2] in each tuple is the name of that method.
         if stack[-2][2] == "script":
-            cmd = "Powershell -NonInteractive -NoProfile -ExecutionPolicy Bypass {}".format(
-                cmd.replace('"', '\\"')
-            )
+            cmd = f'"{shell}" -NonInteractive -NoProfile -ExecutionPolicy Bypass -Command {cmd}'
```

Comment:
Hmm. I know we run Py3.6+ on windows packages, actually 3.7, but salt itself still supports 3.5, which doesn't support f-strings....

Reply:
You're right... coming from SSE, I forget we're still on 3.5